lib/ui/common/recommendation_detail_listitem/recommendation_detail_listitem.dart

```diff
@@ -151,38 +151,12 @@ class RecommendationDetailListItem extends StatelessWidget {
     );
   }
 
-  List<Container> _buildCircles(List<Color> colors) => colors.map(
+  List<Widget> _buildCircles(List<Color> colors) => colors.map(
         (color) => Container(
-              padding: _Constants.circlePadding,
-              child: CustomPaint(
-                painter: _DrawCircle(color: color),
-              ),
-            ),
+          height: _Constants.circleSize,
```

Comment:
could this be dynamic? so it fits the height maybe or at least be a param, I might want bigger circles in some places

Reply:
Done!